src/style.css

```diff
@@ -1 +1,70 @@
 /* Styles go here */
+
+* {
+  margin: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.logo {
+  margin-top: 5px;
+}
+
+.header {
+  max-width: 1200px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav-links {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  font-style: normal;
```

Comment:
Why you add this rule if it doesn't affect anything?

Reply:
ok, thank you, changes are pushed,